task_21/Aleksandr.Timoshenko/src/main/java/com/hillel/tasks/Train.java

```diff
@@ -0,0 +1,82 @@
+package com.hillel.tasks;
+
+public class Train {
+
+  private static final int MAX_SEATS_NUMBERS = 36;
+
+  private int numbersOfCarriage;
+  private String[][] carriage;
```

Comment:
A better OOP solution would be a creation of a Carriage itself, and your train would have a property:
private Carriage[] carriages; i.e. an array of carriages or a list for example, for a better flexibility